scss/editor/_layout.scss

```diff
@@ -89,29 +103,38 @@
         padding: $input-padding-y;
         width: 100%;
         height: 100%;
-        border-width: 1px;
+        border-width: 1px 0 0;
         border-style: solid;
         outline: 0;
     }
 
 
     // Insert table
     .k-ct-popup {
-        padding: $padding-y-sm;
-        // TODO: improve width calculation
-        // width: (1rem + $padding-y-sm) * 8;
-    }
-    .k-ct-cell {
-        margin: $padding-y-sm / 2;
-        width: 1rem;
-        height: 1rem;
         box-sizing: border-box;
-        border: 1px solid;
-        border-color: inherit;
-        display: inline-block;
-        vertical-align: top;
-        overflow: hidden;
-        opacity: .7;
+        width: 190px;
+        padding: 5px;
+
+        .k-editor-toolbar {
+            text-align: center;
+
+            .k-tool-text {
+                display: inline;
+            }
+        }
+
+        .k-ct-cell {
+            margin: $padding-y-sm / 2;
+            width: 20px;
+            height: 20px;
```

Comment:
Is there a relationship between the `.k-ct-popup` width and padding, and the `.k-ct-cell` margin / width and height? Can be extracted in a variable.